### PR TITLE
Registration updates

### PR DESCRIPTION
**Pull request by rowanhogan**
- Add gender & title fields to reg form
- Update form validations per feedback
- Add required field indicators (*)
- Style links for form footers
- Update radio options to be more specific

### CODE REVIEW

File: source/components/RegisterForm/MetadataForm/form.js [hunk 1]
```diff
@@ -97,10 +97,10 @@ export default ({ registration }) => {
       type: 'radio',
       choices: [
         { value: '1 - 50' },
-        { value: '50 - 250' },
-        { value: '250 - 500' },
-        { value: '500 - 1000' },
-        { value: '1000+' }
+        { value: '51 - 250' },
```
Comment (kizza) [inline source/components/RegisterForm/MetadataForm/form.js]: lol. indeed 👍

File: source/components/RegisterForm/MetadataForm/form.js [hunk 2]
```diff
@@ -1,6 +1,14 @@
+import moment from 'moment'
 import merge from 'lodash/merge'
 import * as validators from 'constructicon/lib/validators'
 
+const dateIsAfter = (date = moment().startOf('day'), msg = `Date must be after ${date.format('DD/MM/YYYY')}`) => {
+  return (val) => {
+    const value = moment.isMoment(val) ? val : moment(val)
```
Comment (kizza) [inline source/components/RegisterForm/MetadataForm/form.js]: ...having `val` and `value` is a little weird... but assuming this is copy/pasted from somewhere else where this has been solved :+1:

Comment (rowanhogan) [reply source/components/RegisterForm/MetadataForm/form.js]: Your assumption is correct

File: source/components/RegisterForm/CreatePageForm/form.js [hunk 3]
```diff
@@ -1,58 +1,88 @@
+import moment from 'moment'
 import * as validators from 'constructicon/lib/validators'
 
+const dateIsBefore = (date = moment().startOf('day'), msg = `Date must be before ${date.format('DD/MM/YYYY')}`) => {
```
Comment (kizza) [inline source/components/RegisterForm/CreatePageForm/form.js]: Ahh, now I see this one.  Feels like there could be a helper library that shares the boilerplate of `after/before`.  But life is short :+1:

Comment (kizza) [reply source/components/RegisterForm/CreatePageForm/form.js]: Maybe something to add to constructicon validators "one day"

Comment (rowanhogan) [reply source/components/RegisterForm/CreatePageForm/form.js]: Yeah absolutely! The only reason it hasn't been so far is that it includes adding moment as a dependency... If we're doing that we should add proper datepicker support to Constructicon as well

**Pull request merged**